Client/Frontend/Browser/BrowserViewController/BrowserViewController+WKNavigationDelegate.swift

```diff
@@ -41,7 +41,15 @@ extension BrowserViewController {
     tabManager[webView] ?? (webView as? TabWebView)?.tab
   }
   
-  fileprivate func handleExternalURL(_ url: URL, openedURLCompletionHandler: ((Bool) -> Void)? = nil) {
+  fileprivate func handleExternalURL(_ url: URL,
+                                     isMainFrame: Bool,
+                                     openedURLCompletionHandler: ((Bool) -> Void)? = nil) {
+    
+    // We do not want certain schemes to be opened externally when called from subframes.
+    if ["tel", "sms"].contains(url.scheme) && !isMainFrame {
```

Comment:
@iccub can we use a whitelist here `["http", "https", "data"...]` instead of blocklist? otherwise we may miss some custom schemes like https://codeutility.org/launching-viber-app-via-url-scheme-on-ios-stack-overflow/